android/src/app/organicmaps/util/Utils.java

```diff
@@ -810,4 +811,29 @@ public void onCompleted(final boolean success, @Nullable final String zipPath)
       });
     }
   }
+
+  @Nullable
+  public static <T> T getParcelable(Bundle args, String key, Class<T> clazz)
+  {
+    args.setClassLoader(clazz.getClassLoader());
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU)
+    {
+      return args.getParcelable(key, clazz);
+    }
+    return getParcelableOld(args, key, clazz);
+  }
+
+  @SuppressWarnings("deprecation")
+  @Nullable
+  private static <T> T getParcelableOld(android.os.Bundle args, String key, Class<T> clazz)
```

Comment:
Do we really need a function for that? It is used only in one place.

Reply:
I think it looks better in that way. And Android Studio won't show that `getParcelable` is deprecated.

Example without this function:
<img width="588" alt="Screenshot 2022-10-31 at 13 22 52" src="https://user-images.githubusercontent.com/10351358/199006870-7d319c42-a248-4014-84f6-d4008aa3ad2b.png">

Reply:
The older linter was smarter...